client/my-sites/comments/comment/utils.js

```diff
@@ -0,0 +1,62 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get, includes } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { decodeEntities } from 'lib/formatting';
+
+/**
+ * Return the comment author display name, or "Anonymous" if the comment author has no name.
+ *
+ * @param {Object} comment A comment object.
+ * @returns {String} The comment author display name.
+ */
+export const getAuthorDisplayName = comment =>
+	decodeEntities( get( comment, 'author.name', translate( 'Anonymous' ) ) );
+
+/**
+ * Create a stripped down comment object containing only the information needed by
+ * CommentList's change status and reply functions, and their respective undos.
+ *
+ * @param {Object} comment A comment object.
+ * @returns {Object} A stripped down comment object.
+ */
+export const getMinimalComment = comment => ( {
```

Comment:
Maybe `getBasicCommentData`, `getBasicCommentFields` or something similar? I think `Minimal` could have some sorting connotation. I'll defer to our native English speakers here.

Reply:
you can always get really creative with destructuring and do:

```js
export const getMinimalComment = ( { ID: commentId, i_like: isLiked, post: { ID: postId }, status } ) => ( { commentId, isLiked, postId, status } )
```

Reply:
Thank you both, I'll keep this like it is for now (it's not even used anywhere yet! 🙂 ) and improve on it when the time comes.

Reply:
Careful with deep destructuring folks. I find that folks have trouble reading it if it's more than one level deep. Best to be a little bit more verbose and readable over terse and 🤔.